src/site/twitch.tv/modules/hidden-elements/HiddenElementsModule.vue

```diff
@@ -216,4 +234,23 @@ export const config = [
 		display: none !important;
 	}
 }
+
+.seventv-hide-uptime {
+	span[class="live-time"] {
+		display: none !important;
+	}
+}
+.seventv-hide-viewer-count {
+	p[data-a-target="animated-channel-viewers-count"] {
+		display: none !important;
+	}
+	svg[type="color-text-accessible-red"] {
+		display: none !important;
+	}
+}
+.seventv-hide-squad-stream-button {
+	a[href$="squad"] {
+		display: none !important;
+	}
```

Comment:
Most of these selectors are too broad and would result in content being hidden where it shouldn't be